<ui version="4.0" >
 <class>GroupObjectDialog_q</class>
 <widget class="QWidget" name="GroupObjectDialog_q" >
  <property name="enabled" >
   <bool>true</bool>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>927</width>
    <height>245</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Group</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>2</number>
   </property>
   <property name="spacing" >
    <number>-1</number>
   </property>
   <item row="0" column="0" >
    <layout class="QVBoxLayout" >
     <property name="spacing" >
      <number>0</number>
     </property>
     <item>
      <widget class="QFrame" name="frame16" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape" >
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Sunken</enum>
       </property>
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <widget class="QGroupBox" name="buttonGroup2" >
          <property name="styleSheet" >
           <string notr="true" >QGroupBox {
	margin: 0px;
	padding: 4px;
}</string>
          </property>
          <property name="flat" >
           <bool>true</bool>
          </property>
          <layout class="QGridLayout" >
           <property name="topMargin" >
            <number>2</number>
           </property>
           <property name="bottomMargin" >
            <number>2</number>
           </property>
           <item row="0" column="0" >
            <spacer>
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0" >
              <size>
               <width>31</width>
               <height>26</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="1" >
            <widget class="QToolButton" name="iconViewBtn" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize" >
              <size>
               <width>20</width>
               <height>0</height>
              </size>
             </property>
             <property name="text" >
              <string>I</string>
             </property>
             <property name="checkable" >
              <bool>true</bool>
             </property>
             <property name="checked" >
              <bool>true</bool>
             </property>
             <property name="autoExclusive" >
              <bool>true</bool>
             </property>
             <property name="autoRaise" >
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="2" >
            <widget class="QToolButton" name="listViewBtn" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize" >
              <size>
               <width>20</width>
               <height>0</height>
              </size>
             </property>
             <property name="text" >
              <string>L</string>
             </property>
             <property name="checkable" >
              <bool>true</bool>
             </property>
             <property name="autoExclusive" >
              <bool>true</bool>
             </property>
             <property name="autoRepeatInterval" >
              <number>100</number>
             </property>
             <property name="autoRaise" >
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item rowspan="2" row="0" column="1" >
         <widget class="QStackedWidget" name="objectViewsStack" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Expanding" hsizetype="Preferred" >
            <horstretch>250</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize" >
           <size>
            <width>250</width>
            <height>100</height>
           </size>
          </property>
          <property name="acceptDrops" >
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="WStackPage" />
         </widget>
        </item>
        <item row="0" column="2" >
         <widget class="QLabel" name="textLabel2" >
          <property name="text" >
           <string>Comment:</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QFrame" name="frame7" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize" >
           <size>
            <width>300</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>300</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="frameShape" >
           <enum>QFrame::Box</enum>
          </property>
          <property name="frameShadow" >
           <enum>QFrame::Sunken</enum>
          </property>
          <layout class="QGridLayout" >
           <property name="margin" >
            <number>8</number>
           </property>
           <property name="spacing" >
            <number>8</number>
           </property>
           <item row="2" column="1" >
            <spacer>
             <property name="orientation" >
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType" >
              <enum>QSizePolicy::MinimumExpanding</enum>
             </property>
             <property name="sizeHint" stdset="0" >
              <size>
               <width>20</width>
               <height>2</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="1" >
            <widget class="QLineEdit" name="obj_name" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
               <horstretch>200</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize" >
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="cursor" >
              <cursorShape>ArrowCursor</cursorShape>
             </property>
            </widget>
           </item>
           <item row="0" column="0" >
            <widget class="QLabel" name="textLabel1" >
             <property name="text" >
              <string>Name:</string>
             </property>
             <property name="wordWrap" >
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="2" >
         <widget class="QTextEdit" name="comment" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Expanding" hsizetype="Preferred" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize" >
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="tabChangesFocus" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="2" margin="2" />
 <tabstops>
  <tabstop>obj_name</tabstop>
  <tabstop>comment</tabstop>
 </tabstops>
 <includes>
  <include location="local" >ObjectListView.h</include>
 </includes>
 <resources>
  <include location="MainRes.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>obj_name</sender>
   <signal>textChanged(QString)</signal>
   <receiver>GroupObjectDialog_q</receiver>
   <slot>changed()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comment</sender>
   <signal>textChanged()</signal>
   <receiver>GroupObjectDialog_q</receiver>
   <slot>changed()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>iconViewBtn</sender>
   <signal>clicked()</signal>
   <receiver>GroupObjectDialog_q</receiver>
   <slot>switchToIconView()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listViewBtn</sender>
   <signal>clicked()</signal>
   <receiver>GroupObjectDialog_q</receiver>
   <slot>switchToListView()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
